src/main/web_src/src/components/dollyKjede/KjedeItem.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import styled from 'styled-components'
+import './DollyKjede.less'
+
+export interface KjedeItemProps {
+	index: number
+	selected: boolean
+	disabled: boolean
+	text: string
+	onClick: (page: number) => void
+}
+
+const Button = styled.button`
+	background: transparent;
+	text-decoration: ${p => p.style.textDecoration};
+	cursor: ${p => p.style.cursor};
+	border: none;
+	font-size: 15px;
+	padding: 0 9px 0 9px;
+`
+
+export default ({ index, selected, disabled, text, onClick }: KjedeItemProps) => {
+	const textColor = () => {
+		if (selected) {
+			return 'color-black'
+		} else return disabled ? 'color-grey' : 'color-lightblue'
+	}
+
+	const style = {
+		cursor: selected || disabled ? 'auto' : 'pointer',
+		className: textColor(),
+		textDecoration: selected ? 'none' : 'underline'
+	}
+
+	return (
+		<Button
```

Comment:
Hadde vært fint med en `title` på denne når den er disabled, for å forklare opplegget for brukeren. Noe ala "Du må låse opp lenkede måneder for å gjøre endringer på hver enkelt måned" 🤔  (Ok, dette kan sikkert skrives mye bedre, men det var det jeg kom på i farten 😅 )

Reply:
Gode ideer, skal prøve å fikse :)